lib/nuxeo.js

```diff
@@ -113,8 +116,8 @@ class Nuxeo extends Base {
       }
       doFetch(options.url, fetchOptions)
         .then((res) => {
-          this._activeRequests--;
-          if (!(/^2/.test('' + res.status))) {
+          this._activeRequests += 1;
```

Comment:
That's not recommended anymore with the default eslint configuration we use, and I want to limit the exceptions :)